eras/babbage/formal-spec/frontmatter.tex

```diff
@@ -19,40 +19,50 @@
 \renewcommand{\thepage}{\arabic{page}}
 \setcounter{page}{1}
 
-\title{Formal Specification of the Cardano Ledger with Plutus
-Integration}
+\title{Formal Specification of the Cardano Ledger for the Babbage era}
 
 \author{
-   Polina Vinogradova \\ {\small \texttt{polina.vinogradova@iohk.io}} \\
    Andre Knispel \\ {\small \texttt{andre.knispel@iohk.io}} \\
+   Jared Corduan \\ {\small \texttt{jared.corduan@iohk.io}} \\
    }
 
 \date{}
 
 \maketitle
 
 \begin{abstract}
-This document presents modifications to the Shelley ledger
-specification~\cite{shelley_spec} that will enable it to support the requirements for Plutus Foundation, within the context of the Goguen era
-of Cardano. This specification is based on the Plutus system and the extended
-UTxO specification of the mockchain on which it operates, as outlined in~\cite{chakravarty2020extended}.
-%
-We present a unified way to process both Plutus scripts and Shelley-style multi-signature scripts
-(see~\cite{multi_sig}).
+This document presents the modifications to the Alonzo ledger specification~
+(see \cite{alonzo_spec}) for the Babbage era.
+The Babbage era introduces two main groups of changes.
+
+The first group involves new ways of providing data to Plutus scripts.
+In particular, there is now support for
+\href{https://cips.cardano.org/cips/cip31/}{reference inputs},
+\href{https://cips.cardano.org/cips/cip32/}{inline datums}, and
+\href{https://cips.cardano.org/cips/cip33/}{reference scripts}.
+Additionally, the Babbage era supports
+\href{https://github.com/cardano-foundation/CIPs/pull/216}{collateral outputs},
+which reduces the risk of using collateral inputs for hardware wallets.
```

Comment:
I'd say it supports collateral outputs & asserting the exact collateral. The first helps with managing collateral for all wallets (for example, Nami wouldn't need to create a special collateral output anymore), and the second helps HW wallets as described.